js/pages/samples.js

```diff
@@ -338,20 +338,28 @@ const marshfield = {
         [
             'Marshfield Clinic, Marshfield Center',
             '1000 N. Oak Ave<br>Marshfield, WI 54449',
-            'Monday - Friday: 8:00 a.m.-2:00 p.m. *please ask about early morning or evening appointment availability if needed',
+            'Monday - Thursday: 7:00 a.m. - 4:00 p.m. and Friday 7a.m. - noon',
```

Comment:
Can you confirm this is the correct text? From the document I see `Monday – Thursday: 7:00 a.m. to 4:00 p.m. and Friday 7:00 a.m. to noon.` This is also the case for this [line 347](https://github.com/episphere/connectApp/pull/565/files#diff-086e163ff162b94330737cfb586d27c44a4871d7792612bf21d2791ead9d7c29R347)